neqo-transport/src/recovery.rs

```diff
@@ -318,6 +318,19 @@ impl LossRecovery {
         self.cc.ssthresh()
     }
 
+    #[cfg(test)]
+    pub fn latest_rtt(&self) -> Duration {
+        self.rtt_vals.latest_rtt
+    }
+
+    pub fn smoothed_rtt(&self) -> Option<Duration> {
+        self.rtt_vals.smoothed_rtt
+    }
```

Comment:
The RttVals::rtt accessor contains the logic that you want to use here.  I would do this:

```suggestion
    pub fn rtt(&self) -> Duration {
        self.rtt_vals.rtt()
    }
```

And then you can drop the test-only `latest_rtt`, relying on this.  In the test you have, the RTT is fine.